assets/src/dashboard/components/bookmark-chip/index.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types'; // import styled from 'styled-components';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+
+import { KEYBOARD_USER_SELECTOR, CHIP_TYPES } from '../../constants';
+import { ReactComponent as BookmarkFill } from '../../icons/bookmark-fill.svg';
+import { ReactComponent as BookmarkOutline } from '../../icons/bookmark-outline.svg';
+
+const chipSize = {
+  [CHIP_TYPES.STANDARD]: {
+    container: '40px',
+    icon: {
+      height: '18px',
+      width: '14px',
+    },
+  },
+  [CHIP_TYPES.SMALL]: {
+    container: '32px',
+    icon: {
+      height: '14px',
+      width: '10px',
+    },
```

Comment:
I'm guessing these aren't theme values because changing the size would break the design?

Reply:
exactly that :D 

> Took some liberty of adjusting the icon size to stay at whole numbers, suspect this may change after Sam takes a pass at the spacing, but this is cleaner than how figma has it with the .8px settings.